src/main/java/io/phasetwo/keycloak/magic/auth/MagicLinkAuthenticatorFactory.java

```diff
@@ -100,7 +100,15 @@ public List<ProviderConfigProperty> getConfigProperties() {
         "Toggle whether magic link should be persistent until expired.");
     actionTokenPersistent.setDefaultValue(true);
 
-    return List.of(createUser, updateProfile, updatePassword, actionTokenPersistent);
+    ProviderConfigProperty actionTokenLifeSpan = new ProviderConfigProperty();
+    actionTokenLifeSpan.setType(ProviderConfigProperty.TEXT_TYPE);
```

Comment:
Looks like I didn't submit this in my first review. This should be `STRING_TYPE`, not `TEXT_TYPE`, which is for a textarea.